crypto_sign/falcon-512/META.yml

```diff
@@ -4,7 +4,7 @@ type: signature
 claimed-nist-level: 1
 length-public-key: 897
 length-secret-key: 1281
-length-signature: 666
+length-signature: 752
```

Comment:
It seems strange to me that the (even if just maximum) signature length of Falcon-compressed is larger than the padded format.

Reply:
With the padded format, signature generation retries until the signature size is below some fixed constant and then padded to this constant length. The constant is fixed such that the probability of a retry is fairly low. With the compressed format, there are no retries; you get what you get on the first iteration. Hence, a "compressed" signature _can_ be longer than a padded signature but on average is shorter.

The "compressed" name is a bit of misnomer imo, since both padded and compressed version use the same format for representing data. A padded signature is just a compressed signature with some extra `0`s tacked on.